monorepo/monorepo-ws/apps/update-supergraph-app/src/configuration/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { EnumOfErrorTypes, UpdateSupergraphAppError } from './errors/errors';
+import { Storage } from '@monorepo-ws/supergraph-manager';
+
+export const checkNumber = (imput: string): number => {
```

Comment:
tyhle validace by chtělo udělat jako libku, ať se to neopakuje v každý aplikaci.

ale klidně v novym MR, ať se to už nemotá do tohodle velkýho.